Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,81 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+joint operational direction of the Node.js Technical Steering Committee (TSC)
+and Node.js Foundation Community Committee (CommComm).
+
+## Node.js Admin Repository
+
+The Node.js Admin Repository (nodejs/admin) will serve the central location for
+managing Node.js GitHub Organization administrative activities. Only Node.js
+GitHub Organization owners will have write permissions to the nodejs/admin
+repository.
+
+## Organization Roles
+
+### Owners
+
+The Node.js Foundation Executive Directory, Node.js TSC Director, TSC Chair,
+and Community Committee Chair, Node.js Foundation Executive Director, and
```

Comment:
Typo: remove first `and`

Copy/paste error: Remove `Node.js Foundation Executive Director` as they're already on the list earlier.